docs/cases/cases-overview.asciidoc

```diff
@@ -102,3 +102,45 @@ NOTE: Once a visualization has been added to a case, it cannot be modified or de
   
 [role="screenshot"]
 image::images/cases-open-vis.png[Shows where the Open Visualization option is]
+
+[float]
+[[cases-export-import]]
+== Export and import cases
+
+Cases can be <<cases-export, exported>> and <<cases-import, imported>> as saved objects through the Kibana {kibana-ref}/managing-saved-objects.html[Saved Objects] UI.
+
+IMPORTANT: Before importing a Lens visualization into a space, ensure the visualization's data is present. Without it, the Lens visualization won't work.
+
+[float]
+[[cases-export]]
+=== Export a case
+Use the export option to move cases between different Kibana instances. When you export a case, the case details, user actions, and comments are exported to a newline-delimited JSON (NDJSON) file. During the export process, text string comments, case alerts, and lens visualizations are exported.
+
+NOTE: Exported alerts will only contain indices' names, the alert document ID, and the ID and name of the rule that triggered the alert. Lens visualizations are exported as JSON blobs.
```

Comment:
I could go either way on this but we might just want to say that exported alerts only contain the necessary metadata for the case to reference the alert instead of the entire alert contents instead of outlining each field that is exported.

Reply:
I can see your point and agree with it @jonathan-buttner. Great suggestion!